docker-compose.yml

```diff
@@ -0,0 +1,49 @@
+version: "3"
+services:
+
+  lisk-explorer:
+    build: .
```

Comment:
Why force a local build when we publish official images? If someone wants to build their own images they'll know to change the `image:` line or use the same name we do.